<!--REQUEST_ID=4b19de81-3bff-4f80-b7eb-e73cb15be2a2-->
Failure on x86_64-linux [(full log)](https://logs.nix.ci/?key=nixos/nixpkgs.48675&attempt_id=adcc89fa-afc6-4682-96b1-ba98989ac7a8)

Attempted: qcdnum, xfitter

<details><summary>Partial log (click to expand)</summary><p>

```
[ 16%] Building C object core/utils/CMakeFiles/rootcint.dir/__/clib/src/strlcpy.c.o
[ 16%] Building C object core/utils/CMakeFiles/rootcint.dir/__/clib/src/strlcat.c.o
[ 16%] Linking CXX executable ../../bin/rootcint
[ 16%] Built target rootcint
make: *** [Makefile:152: all] Error 2
builder for '/nix/store/rc2469ifapcyqh45pw9axq4l62rxz0a0-root-5.34.36.drv' failed with exit code 2
cannot build derivation '/nix/store/2d6h2l6rq20ajpa14cmyijrz334w84px-applgrid-1.4.70.drv': 2 dependencies couldn't be built
cannot build derivation '/nix/store/k2i4blnm1xg9mnyjpbiqgcqdximf97cr-apfelgrid-1.0.1.drv': 1 dependencies couldn't be built
cannot build derivation '/nix/store/8zag1a09zydp5iv0wsz50rlibwirwmxw-xfitter-2.0.0.drv': 3 dependencies couldn't be built
error: build of '/nix/store/8zag1a09zydp5iv0wsz50rlibwirwmxw-xfitter-2.0.0.drv' failed
```
</p></details>

--- a/pkgs/development/libraries/physics/apfelgrid/default.nix
+++ b/pkgs/development/libraries/physics/apfelgrid/default.nix
@@ -1,4 +1,4 @@
-{ stdenv, fetchFromGitHub, autoreconfHook, apfel, applgrid, lhapdf, root }:
+{ stdenv, fetchFromGitHub, autoreconfHook, apfel, applgrid, lhapdf, root5 }:
 
 stdenv.mkDerivation rec {
   name = "apfelgrid-${version}";
@@ -12,7 +12,7 @@ stdenv.mkDerivation rec {
   };
 
   nativeBuildInputs = [ autoreconfHook ];
-  buildInputs = [ apfel applgrid lhapdf root ];
+  buildInputs = [ apfel applgrid lhapdf root5 ];
 
   enableParallelBuilding = true;
 

